core/src/main/java/org/wildfly/extras/creaper/core/online/SslOptions.java

```diff
@@ -174,6 +177,14 @@ public Builder trustStoreType(KeyStoreType trustStoreType) {
             return this;
         }
 
+        /**
+         * Errors during hostname verification will be ignored.
+         */
+        public Builder turnOffHostnameVerification() {
```

Comment:
Yes. 
https://github.com/wildfly-extras/creaper/pull/114/commits/e1afe11718f85d2b52f6f0f730744810c824b4d4#diff-e652327a9248c1fde8e727527b1d6d6fL118
SslOnlineManagementClientTest passed but not HttpsOnlineManagementClientTest